include/graphics.h

```diff
@@ -4677,6 +4685,31 @@ extern const u32 gItemIcon_Meteorite[];
 extern const u32 gItemIconPalette_Meteorite[];
 extern const u32 gItemIcon_MagmaEmblem[];
 extern const u32 gItemIconPalette_MagmaEmblem[];
+extern const u32 gItemIcon_ContestPass[];
+extern const u32 gItemIconPalette_ContestPass[];
+extern const u32 gItemIcon_OaksParcel[];
+extern const u32 gItemIconPalette_OaksParcel[];
+extern const u32 gItemIcon_SecretKey[];
+extern const u32 gItemIconPalette_SecretKey[];
+extern const u32 gItemIcon_BikeVoucher[];
+extern const u32 gItemIconPalette_BikeVoucher[];
+extern const u32 gItemIcon_GoldTeeth[];
+extern const u32 gItemIconPalette_GoldTeeth[];
+extern const u32 gItemIcon_CardKey[];
+extern const u32 gItemIconPalette_CardKey[];
+extern const u32 gItemIcon_LiftKey[];
+extern const u32 gItemIconPalette_Key[];
```

Comment:
This is a duplicate and should be removed. See other related comment.

Reply:
I think this was missed.

Reply:
You're right. Fixed!